src/mon/OSDMonitor.cc

```diff
@@ -8648,6 +8661,39 @@ int OSDMonitor::prepare_command_pool_set(const cmdmap_t& cmdmap,
 	ss << "pg_autoscale_bias must be between 0 and 1000";
 	return -EINVAL;
       }
+    } else if (var == "dedup_tier") {
+      int64_t lowtierpool_id = osdmap.lookup_pg_pool_name(val);
+      if (lowtierpool_id < 0) {
+	ss << "unrecognized pool '" << val << "'";
+	return -ENOENT;
+      }
+      const pg_pool_t *tp = osdmap.get_pg_pool(lowtierpool_id);
+      ceph_assert(tp);
+      if (p.is_erasure()) {
```

Comment:
(This is not a comment on this PR, just a general question I have)  Why we are disallowing erasure for a base pool?  Is it because the class operations for refcounting don't work at the moment or something?

Reply:
The reason why I disallow erasure code pool here is that the EC pool does not support objects_read_sync(). Therefore, I should change the current implementation to make EC pool compatible. As s result, I think this can be future work. 
Also, without this, we can use a combination of a base tier (replicated) and a low tier (replicated or EC pool).

Reply:
Got it.